testing/sdk_cmd.py

```diff
@@ -53,10 +53,14 @@ def service_request(
         log_args=log_args,
         log_response=log_response,
         timeout_seconds=timeout_seconds,
-        **kwargs
+        **kwargs,
     )
 
 
+def history_request(retry=False) -> requests.Response:
```

Comment:
I'm not sold on having this in `sdk_cmd`. It seems to be here just because it calls `cluster_request`. We could rather just call this directly from the `get_failure_metrics` function.